### DISCUSSION TITLE
ERRO jpeg: cannot open source image Filename.webp

### DISCUSSION THREAD

**Discussion by Sanookmakmak** [photoprism/photoprism#2652] [Q&A]
While indexing an animated webp file:

```
2022-08-27 14:49:50 ERRO jpeg: cannot open source image Filename.webp
2022-08-27 14:49:49 INFO convert: converting Filename.webp to Filename.webp.jpg (webp)
```
[Feuerwerk.VGA.webp.zip](https://github.com/photoprism/photoprism/files/9438012/Feuerwerk.VGA.zip)

![Feuerwerk VGA](https://user-images.githubusercontent.com/24833757/187031563-2835e644-aa04-46d5-b19f-f26c235adcc6.gif)

**Comment by lastzero** [comment]
Animated Webp? Wouldn't that be WebM?
